tests/lib/rules/regexp-match.js

```diff
@@ -0,0 +1,262 @@
+/**
+ * @fileoverview Rule to flag non-matching identifiers
+ * @author Matthieu Larcher
+ * @copyright 2015 Matthieu Larcher. All rights reserved.
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var eslint = require("../../../lib/eslint"),
+    ESLintTester = require("eslint-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var eslintTester = new ESLintTester(eslint);
+eslintTester.addRuleTest("lib/rules/regexp-match", {
+    valid: [
+        {
+            code: "firstname = \"Matthieu\"",
+            args: [2, "^[a-z]+$"]
```

Comment:
Use `options` instead of `args`. When you do so, you can leave off the initial 2, as it is not used.